### DISCUSSION TITLE
修改IP

### DISCUSSION THREAD

**Discussion by YuQiangZ** [kubeedge/kubeedge#4608] [Q&A]
当修改了K8S集群的IP,Kubeedge的cloudcore和edgecore需要怎么修改？

**Comment by vincentgoat** [comment]
Hi,
Do you mean the master IP address? If so, please update the kubeConfig in the configmap cloudcore.
```
kubeAPIConfig:
  ...
  kubeConfig: /root/.kube/config
  ...
```
